test/sequential/test-inspector-break-e.js

```diff
@@ -21,4 +21,4 @@ async function runTests() {
   assert.strictEqual(0, (await instance.expectShutdown()).exitCode);
 }
 
-runTests().catch((err) => { console.log(err); process.exit(-1); });
```

Comment:
Is this just to make us the same as upstream? I believe that we added this because without it, if the test fails it is often via a timeout with no additional information. I suppose we should just upstream that change though.

Reply:
I believe this was actually a problem before we had unhandled promise rejection handling.  It should correctly print the unhandled rejection and exit with an error now.

Reply:
Looks like it should do the right thing:

```console
> .\Release\node.exe --expose-internals .\test\sequential\test-inspector-break-e.js
[test] Connecting to a child Node process
[test] Testing /json/list
[err] Debugger listening on ws://127.0.0.1:2957/8bb3dc33-f1f0-43be-98f3-f36d7e0e68d0
[err] For help, see: https://nodejs.org/en/docs/inspector
[err]
[err] Debugger attached.
[err]
Error: foo
   at runTests (E:\GitHub\node-chakracore\test\sequential\test-inspector-break-e.js:19:3)
   at Generator.prototype.next (native code)
   at _tickCallback (internal/process/next_tick.js:68:7)
1
> $LASTEXITCODE
1
```